.gitlab/ornl-ci-conda.yml

```diff
@@ -0,0 +1,126 @@
+stages:
+  - setup
+  - build
+  - test
+  - cleanup
+
+
+### MACHINES
+.on_ascent:
+  variables:
+    SITE: "ascent"
+    CONDA_ENV_NAME: "ascent-env"
+    CONDA_WORK_DIR: ${SDK_WORK_DIR_OPEN}/conda
+    CONDA_REQUIREMENTS: .gitlab/conda-requirements-ppc64le.yml
+    PYTHON_MODULE: "python/3.8-anaconda3"
+    CONDA_ARCH: "ppc64le" 
+
+# .on_summit:
+#   variables:
+#     SITE: "summit"
+#     CONDA_ENV_NAME: "summit-env"
+#     CONDA_WORK_DIR: ${SDK_WORK_DIR_ALPINE}/conda
+#     CONDA_REQUIREMENTS: .gitlab/conda-requirements-ppc64le.yml
+#     PYTHON_MODULE: "python/3.8-anaconda3"
+#     CONDA_ARCH: "ppc64le"
+
+
+### SCRIPTS
+.final_steps:
+  script: &finalize
+    - chgrp -fR csc449 ${CONDA_WORK_DIR} || true
+    - chmod -fR 02770 ${CONDA_WORK_DIR}    || true
+
+.job_tags:
+  tags:
+    - $SITE
+    - $RUNNER_TYPE
+
+.conda_setup:
+  stage: setup
+  script:
+    - mkdir -p ${CONDA_WORK_DIR}
+    - test -d ${CONDA_WORK_DIR}/miniconda3-${SITE} && exit 0
+    - wget -q https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-${CONDA_ARCH}.sh -O ${CONDA_WORK_DIR}/miniconda-${SITE}.sh
+    - chmod +x ${CONDA_WORK_DIR}/miniconda-${SITE}.sh
+    - ${CONDA_WORK_DIR}/miniconda-${SITE}.sh -b -p ${CONDA_WORK_DIR}/miniconda3-${SITE}
+    - *finalize
+
+.conda_env_setup:
+  stage: setup
+  script:
+    - test -d ${CONDA_WORK_DIR}/${CONDA_ENV_NAME} && exit 0
+    - module add ${PYTHON_MODULE}
+    - source ${CONDA_WORK_DIR}/miniconda3-${SITE}/etc/profile.d/conda.sh
+    - eval "$(conda shell.posix hook)"
+    - conda create --prefix ${CONDA_WORK_DIR}/${CONDA_ENV_NAME} -y
+    - *finalize
+
+.conda_build:
+  stage: build
+  script:
+    - source ${CONDA_WORK_DIR}/miniconda3-${SITE}/etc/profile.d/conda.sh
+    - eval "$(conda shell.posix hook)"
+    - conda activate ${CONDA_WORK_DIR}/${CONDA_ENV_NAME}
+    - conda env update --file ${CONDA_REQUIREMENTS}
+    - *finalize
+
+.conda_test:
+  stage: test
+  variables:
+    MAINTAINER: "arambula2@llnl.gov"
+    TESTS_GROUP: "conda"
+    SITE_ID: "ornl-$SITE"
+  script:
+    - source ${CONDA_WORK_DIR}/miniconda3-${SITE}/etc/profile.d/conda.sh
+    - eval "$(conda shell.posix hook)"
+    - conda activate ${CONDA_WORK_DIR}/${CONDA_ENV_NAME}
+    - python3 ci/tests/test.py -s
+    - for TEST in flux parsl rp swift-t parsl-flux rp-flux; do
```

Comment:
let's also have tests in sync with requirements
`for TEST in parsl rp; do`

p.s. I guess we can keep `flux` if only it is provided with `modules`

Reply:
Doesn't look like `flux` module is available on lassen, so I'm assuming the same for ascent? I'll push the change. Let me know if you want me to add an explicit `module load <flux>`, and re-enable the test if `flux` is available on Ascent